aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -201,7 +183,7 @@ void div_floor_kernel(TensorIteratorBase& iter) {
                 floordiv += scalar_t(1.0);
               }
             } else {
-              floordiv = copysign(scalar_t(0), a / b);
+              floordiv = std::copysign(scalar_t(0), a / b);
```

Comment:
You can not make this change, it will cause compiler ICE on aarch64 Ubuntu-18.04

Reply:
Renamed to `c10::` for clarity.